docs/testing/sideload-add-in-with-unified-manifest.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Sideload Office Add-ins that use the unified manifest for Microsoft 365
+description: Test your Office Add-in on Windows by sideloading.
+ms.date: 09/19/2024
+ms.localizationpriority: medium
+---
+
+# Sideload Office Add-ins that use the unified manifest for Microsoft 365
+
+The process of sideloading an add-in that uses the [Unified manifest for Microsoft 365](../develop/json-manifest-overview.md) varies depending on the tool you want to use and on how the add-in project was created. 
+
+> [!NOTE]
+> An add-in that uses the unified manifest can be sideloaded on Office on Windows, Version 2304 (Build 16320.20000) or later. Currently, it cannot be sideloaded on the web, Mac, or iPad.
```

Comment:
```suggestion
> An add-in that uses the unified manifest can be sideloaded on Office on Windows, Version 2304 (Build 16320.20000) or later. Currently, it can't be sideloaded on the web, Mac, or iPad.
```